File: common-app/src/main/java/com/lxd/common_app/network/ApiService.java
package com.lxd.common_app.network;

import com.lxd.common_app.model.MovieEntity;

import io.reactivex.Observable;
import retrofit2.http.HTTP;
import retrofit2.http.Query;

/**
 * Created by Liuxd on 2016/9/23 11:27.
 */

public interface ApiService {
    @HTTP(path = "https://api.douban.com/v2/movie/top250", method = "GET")
    Observable<MovieEntity> getMovie(@Query("start") int start, @Query("count") int count);

}

File: jsoup-study/src/main/java/com/asura/jsoup_study/api/ApiService.kt
package com.asura.jsoup_study.api

import com.asura.jsoup_study.config.Constant
import io.reactivex.Observable
import retrofit2.http.GET
import retrofit2.http.Path


/**
 * @author Created by Asura on 2018/6/8 14:19.
 */
interface ApiService {
    @GET(value = Constant.URL_NEW_GIRL)
    fun getBeautifulGirl(@Path("page") page: Int): Observable<String>;
}